app/graphql/mutations/create_response_mutation.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Mutations
+  class CreateResponseMutation < BaseMutation
+    description "Create a new survey's response"
+
+    argument :response, Types::Inputs::ResponseInput, required: true
+
+    field :success, Boolean, null: false
+
+    def resolve(response:)
+      response_form = ResponseForm.new(response)
+
+      raise(GraphQL::ExecutionError, response_form.errors.full_messages.to_sentence) unless response_form.save
+
+      {
+        success: true
+      }
```

Comment:
Feels a bti awkward to just return `success: true`. In the REST API, we just send 201 so I understand where this comes from but still returning the actual data saved seems more standard as per the [official doc](https://graphql.org/learn/queries/#mutations).